tests/test_strds_area_stats_univar.py

```diff
@@ -16,6 +16,12 @@
 __maintainer__ = "Soeren Gebbert"
 __email__ = "soerengebbert@googlemail.com"
 
+# TODO use modis data
```

Comment:
Is this comment still valid since "modis_lst" is used below?

Reply:
I tried it but it was not working. So I let the comment there